Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDocumentLayout.h

```diff
@@ -109,6 +109,7 @@ class PDFDocumentLayout {
         PageGeometries = 1 << 0,
         DocumentBounds = 1 << 1,
     };
+    static constexpr OptionSet<LayoutUpdateChange> allLayoutChangeTypes { LayoutUpdateChange::PageGeometries, LayoutUpdateChange::DocumentBounds };
```

Comment:
Maybe this can just be moved next to where it's used? I only see one use.

Reply:
Makes sense, I'll do that.

Reply:
Fixed in [325344f](https://github.com/WebKit/WebKit/commit/325344f7bffddedffc0d37bb8458589152723cfc)